libs/utils/utils.js

```diff
@@ -656,6 +656,21 @@ export function decorateLinks(el) {
         window.adobeIMS?.signIn();
       });
     }
+    const copyEvent = '#_evt-copy-link';
+    if (a.href.includes(copyEvent)) {
+      const link = a.href.split('#_evt-copy-link')[0];
+      const isConButton = ['EM', 'STRONG'].includes(a.parentElement.nodeName) || a.classList.contains('con-button');
+      if (!isConButton) a.classList.add('static', 'copy-link');
+      a.href = '';
+      if (navigator.share) {
+        a.addEventListener('click', async (e) => {
+          e.preventDefault();
+          await navigator.share({ title: link, url: link });
+        });
```

Comment:
Do we not break the links on firefox? It seems it does not have any support for it at all.

Reply:
Yes, this is not supported by some desktop browsers. However, this is a mobile-only feature and the API is supported for all the major mobile browsers.

Reply:
See https://github.com/adobecom/milo/pull/2804#pullrequestreview-2272059085 - I think the default fallback behavior is wrong, an empty href leads to a refresh.

Reply:
Ideally this issue should not be seen because the mobile-specific case would be handled by mep and hence, we should not see the block ( with `#_evt-copy`) on browsers where the API is not supported. Nonetheless, I do agree that we need to have a proper fallback and had asked this question on jira.
To prevent the refresh, I can simply add the if check inside `addEventListener` after `e.preventDefault()`. This would prevent the refresh, but the click behavior of the link won't function.
@mokimo What do you think about this? Or do you suggest to redirect to the original link wherever this is not supported?

Reply:
Yes, I think we should handle this case. Even though _you_ intend that this feature is used on mobile through MEP, it exists and can be authored. 
We should ensure this works in any case or remove the element in cases when it suddenly appears (desktop) when it should not, guarding authors from using this feature in an unintended way.

Reply:
Sure, let me remove the element in case `navigator.share()` is not supported. Thank you for your input on this.